src/algebra/module.lean

```diff
@@ -111,6 +111,8 @@ instance semiring.to_semimodule [r : semiring α] : semimodule α α :=
 instance ring.to_module [r : ring α] : module α α :=
 { ..semiring.to_semimodule }
 
+attribute [instance, priority 1500] ring.to_module
```

Comment:
Can this be done with `@[priority 1500]` on line 111 instead? I think this is what Floris did in #1724 .

Are there explicitly documented guidelines for how to choose these priorities? If not we should write them and make them easy to find.

Reply:
In fact this line was needed before #1724 to guarantee that Lean would consider this instance before silly ones that always apply. Since #1724, this is not needed any more, so I have just removed this line.